test/jdk/com/sun/net/httpserver/simpleserver/CustomFileSystemTest.java

```diff
@@ -71,6 +71,7 @@
 import com.sun.net.httpserver.SimpleFileServer.OutputLevel;
 import jdk.test.lib.Platform;
 import jdk.test.lib.net.URIBuilder;
+import jtreg.SkippedException;
```

Comment:
Same remark here: 
You should use the testng exception instead - testng doesn't recognize the jtreg exception